hie-plugin-api/Haskell/Ide/Engine/Cradle.hs

```diff
@@ -398,7 +406,8 @@ cabalHelperCradle file = do
         getComponent env (toList units) relativeFp
           >>= \case
             Just comp -> do
-              let fs = getFlags comp
+              let fs' = getFlags comp
+              let fs = map (fixImportDirs root) fs'
```

Comment:
What is `root` here? Is it where the `cabal.project` file is? If so, it's not right.

Reply:
root ist the root of the cradle. E.g. for hie-plugin-api, this ought to be: `<path-to-hie>/hie-plugin-api`